packages/noya-state/src/reducers/canvasReducer.ts

```diff
@@ -535,6 +538,43 @@ export function canvasReducer(
 
             break;
           }
+          case 'addSymbolLayer': {
```

Comment:
We should move `'addSymbolLayer'` into the `canvasReducer`, since it doesn't depend on the interaction state. I.e. we can define the action next to `'addShapePathLayer'`, and we can move this `case` into the top-level `switch` of the canvasReducer. 

That should simplify things a little, and it'll also let us move the `uuid()` call directly into the canvasReducer, so we can remove the `id` parameter from the action